lib/ui/pages/landing_page/landing_page.dart

```diff
@@ -188,6 +188,7 @@ class _LandingPageState extends ConsumerState<LandingPage> {
             height: 10,
           ),
           CustomActionButton(
+            key: const ValueKey('NewImageActionButton'),
```

Comment:
Does it make sense to create a file where we store all key strings?

Reply:
Thats a good idea!